source/iostreams.tex

```diff
@@ -4585,13 +4585,11 @@
 \tcode{setstate(failbit | eofbit)}
 (which may throw
 \tcode{ios_base::failure}).
+After any preparation is completed, \exposid{ok_} is set to the value of \tcode{is.good()}.
```

Comment:
As this describes the final step, I think it would make more sense to be right at the end, after the "During preparation" sentence (maybe as a separate paragraph).

That way the structure would be:

- The function extracts and discards each whitespace character.
- If extracting a character returns eof, set `failbit|eofbit`.
- Whitespace is determined using `ctype`.
- The constructor may set `failbit` and throw. _(for ... reasons? I guess the footnote covers that)_
- After any preparation is completed, set `ok_`.

What do you think?